### DISCUSSION TITLE
How do I configure LibreChat and an ALB such that a prescribed URL correctly forwards to LibreChat?

### DISCUSSION THREAD

**Discussion by itrase** [danny-avila/LibreChat#736] [Q&A]
This is likely a very basic networking question.
I can set up an ALB on AWS set up such that requests to https://xx.xyz.com/ forward to LibreChat (also accessible via direct IP at http://xx.yy.zz.ww:3080). This works perfectly well.
However, what I want to do is forward requests from https://xx.xyz.com/chat-app to LibreChat. When I try to set this up, I get a blank web page, as it looks like the HTML cannot find the /assets directory. Is there something I need to change in the LibreChat configuration to enable this, or is this an ALB issue?

**Comment by danny-avila** [comment]
I don't know what ALB is but I think this can easily done with NGINX: https://chat.openai.com/share/5d8e886a-29d6-45dc-8098-23a3621ad8dd
